client/hosting-overview/components/active-domains-card.tsx

```diff
@@ -15,7 +16,7 @@ const ActiveDomainsCard: FC = () => {
 	const translate = useTranslate();
 
 	return (
-		<Card className="hosting-overview__card">
+		<Card className={ classNames( 'hosting-overview__card', 'hosting-overview__active-domains' ) }>
```

Comment:
We don't need `classnames` for this:
```suggestion
		<Card className="hosting-overview__card hosting-overview__active-domains">
```

Reply:
I use it because this way it's easier to see that there are multiple classnames, do you think it needs to be removed?

Reply:
I was just concerned about increasing the bundle size more than necessary because of the `classnames` import but I guess https://github.com/Automattic/wp-calypso/pull/89584#issuecomment-2059375707.